week1/homework.md

```diff
@@ -0,0 +1,59 @@
+# Homework week 1
+
```

Comment:
I'd add:
1) that this first part of the homework is mandatory
2) what's the objective of this part, something like "The idea is to fork the homework repository from hack your future, clone it into your computer, do some changes, and push the changes into your fork (copy) of the repository."

Reply:
I dont think we should specify when stuff is mandatory. Only specify when it is optional.